Documentation/EyeTracking/EyeTracking_BasicSetup.md

```diff
@@ -43,59 +43,83 @@ You can simply select _DefaultMixedRealityToolkitConfigurationProfile_ and then
 
 ### Accessing eye gaze data
-Now that your scene is set up to use Eye Tracking, let's take a look at how to access it in your scripts: 
-[Accessing Eye Tracking Data in your Unity Script](EyeTracking_EyeGazeProvider.md).
+Now that your scene is set up to use eye tracking, let's take a look at how to access it in your scripts: 
+[Accessing eye tracking data via EyeGazeProvider](EyeTracking_EyeGazeProvider.md) and [eye-supported target selections](EyeTracking_TargetSelection.md).
 
  
 ### Testing your Unity app on a HoloLens 2
-Building your app with Eye Tracking should be similar to how you would compile other HoloLens 2 MRTK apps. 
+Building your app with eye tracking should be similar to how you would compile other HoloLens 2 MRTK apps. 
 The only difference is that the *'Gaze Input'* capability is unfortunately not yet supported by Unity under 'Player Settings -> Publishing Settings -> Capabilities'. 
-To use Eye Tracking on your HoloLens 2 device, you need to manually edit the package manifest that is part of your build Visual Studio project.
-Follow these steps:
+To use eye tracking on your HoloLens 2 device, you need to manually edit the package manifest that is part of your built Visual Studio project.
 
+Follow these steps:
 1. Build your Unity project as you would normally do for _HoloLens 2_.
 2. Open your compiled Visual Studio project and then open the _'Package.appxmanifest'_ in your solution.
-3. Make sure to tick the _'Gaze Input'_ checkbox under _Capabilities_.
+3. Make sure to tick the _'GazeInput'_ checkbox under _Capabilities_.
 
 ![Enabling Gaze Input in Visual Studio](../Images/EyeTracking/mrtk_et_gazeinput.jpg)
 
-**IMPORTANT:** Don't forget to calibrate HoloLens for the current user. Without calibration, eye tracking won't work. Calibration can be found under system settings.  
+You don't see a _'GazeInput'_ capability? 
+   - Check that your system meets the [prerequisites for using MRTK](../GettingStartedWithTheMRTK.md) (in particular the Windows SDK version). 
+   - You can also manually add the entry by opening the appxmanifest in an XML editor and adding the following:
+
+```xml
+  <Capabilities>
+    <DeviceCapability Name="gazeInput" />
+  </Capabilities>
+```
+
+#### Have you eye calibrated?
+Finally, please don't forget to run through the eye calibration on your HoloLens 2. 
+The eye tracking system will not return any input if the user is not calibrated. 
+Easiest way to get to the calibration is by flipping up the visor and back down. 
+A system notification should appear welcoming you as a new user and asking you to go through the eye calibration.
+Alternatively you can find the eye calibration in the system settings: Settings -> System -> Utilities -> Open Calibration.
+
+#### Do you see the permission prompt?
+When starting the app on your HoloLens 2 for the first time, a prompt should pop up asking the user for permission to use eye tracking. 
```

Comment:
Question unrelated to this, but I haven't tested this myself so I'm curious what the answer is - if I accidentally press no, does that apply forever (in which case, i guess I have to go to settings to enable things for the app right?). If that's the case, it might be worth adding that, otherwise people might not know what to do to recover from that very easy accident (i.e. if my index finger tracking is just off and I press no that's bad)

Reply:
Yep, it's similar to the microphone privacy settings. You would have to go to the settings page where you see which apps got permissions to use your mic or in this case your eye tracker.